style.css

```diff
@@ -1,66 +1,78 @@
-
 header {
   background-color: brown;
-  color:white;
+  color: white;
   font-size: 2em;
   text-align: center;
   width: auto;
 }
 
 h2 {
-    font-size: 1.5em;
-    color: rgb(104, 80, 47);    
+  font-size: 1.5em;
+  color: rgb(104, 80, 47);
 }
 
 .important {
-    color: rgb(134, 99, 54);
-    font-weight: bold;
+  color: rgb(134, 99, 54);
+  font-weight: bold;
+}
+
+section,
+article {
+  float: left;
+  width: 60%;
 }
 
-section, article {
-    float: left;
-    width: 60%;
+aside {
+  float: right;
+  width: 30%;
+  text-align: center;
 }
-  
-nav{
-    border-bottom:2px inset brown;
+
+button.reservation {
+  background-color: brown;
+  text-transform: capitalize;
+  border-radius: 5px;
 }
 
-  nav ul {
+nav {
+  border-bottom: 2px inset brown;
+}
+
+nav ul {
   display: flex;
   justify-content: space-around;
   list-style: none;
 }
 
-nav a{
-    text-decoration: none;
-    color: brown;
-    background-color: white;
-    font-size: 1.5em;
+nav a {
+  text-decoration: none;
+  color: brown;
+  background-color: white;
+  font-size: 1.5em;
 }
 
 nav li {
   display: inline;
 }
 
-  nav a:hover {
```

Comment:
Everyone, try to avoid adding such stylistic change in format as it makes the commit harder to read than it should be. All those two spaces add a lot of noise.